bip-taro-addr.mediawiki

```diff
@@ -79,7 +111,7 @@ program, as specified in BIP 341, using the included key as the internal key.
 
 With the target taproot public key script constructed, the asset is sent to the
 receiver with the execution of the following steps:
-# Construct a valid transaction that spends an input that holds the referenced <code>asset_id</code> and ''at least'' <code>amt</code> units of the asset.
+# Construct a valid transaction that spends an input that holds the referenced <code>asset_id</code> and ''exactly'' <code>amt</code> units of the asset.
```

Comment:
This is also important: otherwise the receiver wouldn't be able to reconstruct the derived output on chain.

Reply:
Isn't this at least, where the output for the receiver is 'exactly' amt and the change output carries the remainder? As specified below.